clients/src/main/java/org/apache/kafka/common/requests/UpdateMetadataRequest.java

```diff
@@ -123,14 +129,14 @@ public UpdateMetadataRequest(int controllerId, int controllerEpoch, Set<BrokerEn
     }
 
     /**
-     * Constructor for version 1.
+     * Constructor for version 2.
```

Comment:
We will still need a separate constructor for v1 of UpdateMetadataRequest since in ControllerChannelManager, we may need to send a v1 request depending on inter.broker.protocol.